source/assets/sass/visual-design/_typography.scss

```diff
@@ -18,29 +18,6 @@ $sl-font-weights: (
   }
 }
 
-$sl-font-size--x-small: functions.sl-px-to-rem(14px) !default;
-$sl-font-size--small: functions.sl-px-to-rem(16px) !default;
-$sl-font-size--medium: functions.sl-px-to-rem(18px) !default;
-$sl-font-size--large: functions.sl-px-to-rem(20px) !default;
-$sl-font-size--x-large: functions.sl-px-to-rem(24px) !default;
-$sl-font-size--xx-large: functions.sl-px-to-rem(28px) !default;
-$sl-font-size--xxx-large: functions.sl-px-to-rem(32px) !default;
-$sl-font-size--xxxx-large: functions.sl-px-to-rem(80px) !default;
-$sl-font-sizes: (
-  small: $sl-font-size--small,
-  medium: $sl-font-size--medium,
-  large: $sl-font-size--large,
-  x-large: $sl-font-size--x-large,
-  xx-large: $sl-font-size--xx-large,
-  xxx-large: $sl-font-size--xxx-large,
-  xxxx-large: $sl-font-size--xxxx-large,
-);
-
-@each $name, $value in $sl-font-sizes {
-  .sl-font-size--#{$name} {
-    font-size: $value;
-  }
-}
```

Comment:
I didn't see where this class is being used (as a utility class) so I removed it.